Sources/Logging.php

```diff
@@ -249,10 +249,10 @@ function displayDebug()
 	$warnings = 0;
 	if (!empty($db_cache))
 	{
-		foreach ($db_cache as $q => $query_data)
+		foreach ($db_cache as $q => $qq)
```

Comment:
On my pr you mention that qq is bad and
now it's good?
https://github.com/SimpleMachines/SMF2.1/pull/4472#discussion_r162849616

Reply:
No, I just missed this when reimplementing those changes. :)